src/components/CompanyMatchPage/SearchBarCont.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import {Form,FormControl, Button} from 'react-bootstrap'
+const SearchBarCont = () => {
```

Comment:
This file should be in the components folder.

Reply:
This component only related to CompanyHome page so i think its good to be there